samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1430,6 +1430,106 @@
           "Resource": "*"
         }]
       }
+    },
+    "DecodeSSMParameterPolicy": {
+      "Description": "Gives access to SSM key and parameter to load secrets in this account.",
+      "Parameters": {
+        "ParameterName": {
+          "Description":"The name of the secret stored in SSM in your account."
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ssm:DescribeParameters"
+            ],
+            "Resource": "*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ssm:GetParameters"
```

Comment:
I'm reviewing [SSM actions](https://docs.aws.amazon.com/systems-manager/latest/userguide/auth-and-access-control-permissions-reference.html) and it looks like there is a `ssm:GetParameter` option that only allows accessing one parameter instead of several. Is there a reason why you chose to use `ssm:GetParameters` over `ssm:GetParameter`?

Reply:
I was basing this off of a tutorial example that used `ssm:GetParameters` instead of `ssm:GetParameter`. I'll look into seeing what the difference is, because it didn't specify.

Reply:
I *think* `ssm:GetParameters` allows the use of wildcards in the parameter names, but I don't know if how/if it differs from `ssm:GetParameter` besides that. I've resolved it by having both of them in the template.